#Message definition to set the position of all the joints of 1 arm in the humanoid robot
#(4 joints in each arm, 2 at the shoulder and 2 at the elbow) 
uint8 id1
uint8 id2
uint8 id3
uint8 id4
string item1
string item2
string item3
string item4
int32 value1
int32 value2
int32 value3
int32 value4



